linera-core/src/client/mod.rs

```diff
@@ -1984,15 +1982,15 @@ where
                 ExecuteBlockOutcome::Executed(certificate) => {
                     return Ok(ClientOutcome::Committed(certificate));
                 }
+                ExecuteBlockOutcome::WaitForTimeout(timeout) => {
+                    return Ok(ClientOutcome::WaitForTimeout(timeout));
+                }
```

Comment:
(Why the order change?)

Reply:
Personal preference - it's IMHO better to group logic by importance: either ged the easy ones out of the way immediately or put them at the end. The `info!` on `Conflict` is the least important branch/case.